### PR TITLE
Add full-flow Oryx text for Python

### PR DESCRIPTION
**Pull request by jungaretti**
Adds a full-flow Oryx text for Python using a sample project.

### CODE REVIEW

File: src/universal/test-project/test.sh [hunk 1]
```diff
@@ -134,6 +134,10 @@ check "run-puppeteer" node puppeteer.js
 # Check Oryx
 check "oryx" oryx --version
 
+# Make sure that Oryx builds Python projects correctly
+check "oryx-build-python" oryx build --property python_version='3.10.4' --property packagedir='/home/codespace/.local/lib/python3.10/site-packages' ./sample/python
```
Comment (samruddhikhandale) [inline src/universal/test-project/test.sh:138]: Even my first instance was to write a similar test.
But now that I am thinking, should  we also write a test to repro the `universal` images' post create command.

https://github.com/devcontainers/images/blob/main/src/universal/.devcontainer/devcontainer.json#L104

This particular command is something that is executed by oryx

File: src/universal/test-project/test.sh [hunk 2]
```diff
@@ -134,6 +134,10 @@ check "run-puppeteer" node puppeteer.js
...
+check "oryx-build-python-result" python3 ./sample/python/src/solve.py
```
Comment (samruddhikhandale) [inline src/universal/test-project/test.sh:139]: Nice test. 

💡 We could also test if oryx worked with `pip list | grep mpmath`

File: src/universal/test-project/test.sh (hunk 1 above)
Comment (samruddhikhandale) [inline src/universal/test-project/test.sh:138]:
```suggestion
pythonVersion=$(python -V 2>&1 | grep -Po '(?<=Python )(.+)')
pythonSite=`python -m site --user-site`
check "oryx-build-python" oryx build --property python_version="${pythonVersion}" --property packagedir="${pythonSite}" ./sample/python
```

File: src/universal/test-project/test.sh (hunk 2 above)
Comment (jungaretti) [reply src/universal/test-project/test.sh:141]: Good idea, I'll add that

File: src/universal/test-project/test.sh (hunk 1 above)
Comment (samruddhikhandale) [reply src/universal/test-project/test.sh]: Not needed. Discussed offline

**Review by samruddhikhandale — APPROVED** [review]
Thanks! ✨  The smoke test for `universal` is supposed to fail.

It is fixed in https://github.com/devcontainers/images/pull/178

**Comment by Musti331** [comment]
₺180

**Comment by Musti331** [comment]
#180

**Comment by Musti331** [comment]
@@ -134,6 +134,10 @@

**Pull request merged**